epytope/tutorials/PolymorphicEpitopePrediction.ipynb

```diff
@@ -88,12 +96,12 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/Users/antonia/PycharmProjects/Kfred/epytope/Core/Generator.py:57: UserWarning: For NM_001190266 bp does not match ref of assigned variant Variant(g.234183368A>G). Pos 645, var ref A, seq ref C \n",
+      "/Users/jonas/Desktop/epytope/epytope/Core/Generator.py:57: UserWarning: For NM_001190266 bp does not match ref of assigned variant Variant(g.234183368A>G). Pos 645, var ref A, seq ref C \n",
```

Comment:
Should we get rid of local paths here?

Reply:
Did not find a way in the last 15min. But I won't focus on that minor detail since it was already there..